src/cmds/net/arping.c

```diff
@@ -110,9 +110,9 @@ int main(int argc, char **argv) {
 		return -EINVAL;
 	}
 
-	strncpy(dst_b, inet_ntoa(dst), sizeof(dst_b));
+	strncpy(dst_b, inet_ntoa(dst), sizeof(dst_b) - 1);
```

Comment:
it should set '\0' at the last byte of the buffer after calling strncpy
`dst_b[sizeof(dst_b) - 1] = '\0';`